integration/constants.go

```diff
@@ -13,8 +13,8 @@ const (
 	StartPortWalletExtensionUnitTest = 38000
 	StartPortFaucetUnitTest          = 42000
 	StartPortFaucetHTTPUnitTest      = 48000
-	StartPortTenscanUnitTest         = 52000
-	StartPortTenGatewayUnitTest      = 56000
+	StartPortTenscanUnitTest         = 12000
```

Comment:
I noticed that tenscan test failing. Grasping on straws here, I thought of just changing the port.